### PR TITLE
Add amazon load balancer actions

### PR DESCRIPTION
**Pull request by Ladas**
Add amazon load balancer actions

### CODE REVIEW

File: app/models/manageiq/providers/amazon/network_manager/load_balancer.rb [hunk 1]
```diff
@@ -1,2 +1,48 @@
 class ManageIQ::Providers::Amazon::NetworkManager::LoadBalancer < ::LoadBalancer
+  def self.raw_create_load_balancer(load_balancer_manager, load_balancer_name, options = {})
+    load_balancer_manager.with_provider_connection(:service => :ElasticLoadBalancing) do |service|
+      _load_balancer = service.client.create_load_balancer(:load_balancer_name => load_balancer_name,
```
Comment (durandom) [inline app/models/manageiq/providers/amazon/network_manager/load_balancer.rb]: `_load_balancer` is not needed?!

File: app/models/manageiq/providers/amazon/network_manager/load_balancer.rb [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+                                                           :security_groups    => options[:security_groups])
+      service.client.register_instances_with_load_balancer(:load_balancer_name => load_balancer_name,
+                                                           :instances          => options[:vms])
+      service.client.configure_health_check(:load_balancer_name => load_balancer_name,
+                                            :health_check       => options[:load_balancer_health_checks])
+    end
+    return load_balancer_name
+  rescue => err
+    _log.error "load_balancer=[#{load_balancer_name}], error: #{err}"
+    raise MiqException::MiqLoadBalancerProvisionError, err.to_s, err.backtrace
```
Comment (durandom) [inline app/models/manageiq/providers/amazon/network_manager/load_balancer.rb]: Looks like these Exceptions havent been merged yet

**Comment by durandom** [comment]
Is this a :chicken: :egg: PR where https://github.com/ManageIQ/manageiq/issues/10976 need this to be merged and the other way round?

**Comment by Ladas** [comment]
@durandom  when I do specs, we will definitely need the exceptions in place. https://github.com/ManageIQ/manageiq/pull/10975/files this is the PR containing the exceptions

**Comment by Ladas** [comment]
@durandom the PR with exceptions was merged

**Comment by durandom** [comment]
kicking build

**Comment by durandom** [comment]
@Ladas does this need to be in euwe?

**Comment by Ladas** [comment]
@durandom good question, I don't know. @blomquisg do we plan LB provisioning in some minor Euwe release?

**Comment by Ladas** [comment]
@miq-bot assign @blomquisg

File: app/models/manageiq/providers/amazon/network_manager/load_balancer.rb (hunk 1 above)
Comment (Ladas) [inline app/models/manageiq/providers/amazon/network_manager/load_balancer.rb]: yes, that is why there is _ for now

**Comment by Ladas** [comment]
@blomquisg should be ready

File: app/models/manageiq/providers/amazon/network_manager/load_balancer.rb (hunk 2 above)
Comment (Ladas) [inline app/models/manageiq/providers/amazon/network_manager/load_balancer.rb]: merged now

**Comment by blomquisg** [comment]
@Ladas can you respond to @durandom's review ... it's leaving a red "X" on this PR.

**Comment by durandom** [comment]
merged, because https://github.com/ManageIQ/manageiq/pull/10975 is merged.

@Ladas @blomquisg I guess this is euwe/yes because :point_up: PR is euwe/yes ?

**Comment by durandom** [comment]
:( reverted in https://github.com/ManageIQ/manageiq-providers-amazon/pull/64

**Pull request merged**